Blocks.vue

```diff
@@ -104,7 +104,7 @@ export default {
     async getCurrentBlock() {
       const fetchBlockId = this.makeBlockId(this.currentBlock - 1);
       if(!(fetchBlockId in this.blockCache)) {
-        const block = await axios.get(this.ledgerBlockService, {
+        const block = await httpClient.get(this.ledgerBlockService, {
           params: {
```

Comment:
I think it should be `searchParams` instead of `params`.

Reply:
THANK YOU! Wow, I can't believe I missed that one: https://github.com/digitalbazaar/bedrock-vue-web-ledger/pull/7/commits/68435c9ae315c63ee8003ea61448e4d93ca11894